src/plugins/funded-loans-mixin.js

```diff
@@ -0,0 +1,39 @@
+import _filter from 'lodash/filter';
+import _get from 'lodash/get';
+import numeral from 'numeral';
+
+export default {
```

Comment:
I think this would make more sense as just a util file that exports 2 functions rather than as a mixin, since there's nothing vue related in this code, but this is fine for getting filtering working.